### PR TITLE
Add more columns for tkctl get tikv

### PR DESCRIPTION
**Pull request by Yisaer**
### What problem does this PR solve? 
add storeid column for tkctl when type get tikv
https://github.com/pingcap/tidb-operator/issues/564

### What is changed and how does it work?
Add type alias for `Tikv v1.Pod` and registered 2 new printer handlers so that `tkctl` will only add `STOREID` column when `tkctl get tikv` only

### Check List 

Tests 

 - Manual test

Code changes

 - Has Go code change

Side effects

 - N/A

Related changes

 - N/A
### Does this PR introduce a user-facing change?:
 
 ```release-note
`tkctl get tikv` add `STOREID` column to show store id for each tikv pod
 ```

### CODE REVIEW

File: pkg/tkctl/readable/printers.go [hunk 1]
```diff
@@ -30,7 +31,11 @@ import (
 )
 
 const (
-	unset = "<none>"
+	unset          = "<none>"
+	kindPD         = "pd"
+	kindTiKV       = "tikv"
+	kindTiDB       = "tidb"
+	componentLabel = "app.kubernetes.io/component"
```
Comment (onlymellb) [inline pkg/tkctl/readable/printers.go]: These constants have been defined [here](https://github.com/pingcap/tidb-operator/blob/master/pkg/label/label.go).

File: pkg/tkctl/readable/printers.go [hunk 2]
```diff
@@ -350,3 +386,11 @@ func translateTimestampSince(timestamp metav1.Time) string {
 
 	return duration.HumanDuration(time.Since(timestamp.Time))
 }
+
+// extra Component Tikv Data
+func extraTikvDataColumn(pod *v1.Pod) *TikvExtraInfoColumn {
+	storeId := pod.Labels["tidb.pingcap.com/store-id"]
```
Comment (onlymellb) [inline pkg/tkctl/readable/printers.go]: ditto

**Comment by Yisaer** [comment]
Here is the result the `tkctl get tikv` will look lile for this PR.
```bash
$ ./tkctl get tikv
NAME          READY   STATUS    MEMORY          CPU             RESTARTS   AGE    STOREID
tidb-tikv-0   1/1     Running   <none>/<none>   <none>/<none>   1          8d     4
tidb-tikv-1   1/1     Running   <none>/<none>   <none>/<none>   1          8d     1
tidb-tikv-2   1/1     Running   <none>/<none>   <none>/<none>   1          8d     5
tidb-tikv-3   1/1     Running   <none>/<none>   <none>/<none>   0          16h    1001
tidb-tikv-4   1/1     Running   <none>/<none>   <none>/<none>   0          16h    1002
tidb-tikv-5   1/1     Running   <none>/<none>   <none>/<none>   0          12h    1003
tidb-tikv-6   1/1     Running   <none>/<none>   <none>/<none>   0          164m   1006
tidb-tikv-7   1/1     Running   <none>/<none>   <none>/<none>   0          164m   1037
tidb-tikv-8   1/1     Running   <none>/<none>   <none>/<none>   0          163m   1038
```

**Comment by gregwebs** [comment]
@aylei PTAL

File: pkg/tkctl/readable/printers.go [hunk 3]
```diff
@@ -73,6 +79,16 @@ func AddHandlers(h printers.PrintHandler) {
 	}
 	h.TableHandler(commonPodColumns, printPod)
 	h.TableHandler(commonPodColumns, printPodList)
+	//tikv columns
+	tikvPodColumns := commonPodColumns
+	storeId := metav1beta1.TableColumnDefinition{
+		Name:        "StoreId",
+		Type:        "string",
+		Description: "Tikv Store Id",
```
Comment (weekface) [inline pkg/tkctl/readable/printers.go]:
```suggestion
		Description: "TiKV StoreID",
```

File: pkg/tkctl/readable/printers.go [hunk 4]
```diff
@@ -73,6 +79,16 @@ func AddHandlers(h printers.PrintHandler) {
 	}
 	h.TableHandler(commonPodColumns, printPod)
 	h.TableHandler(commonPodColumns, printPodList)
+	//tikv columns
+	tikvPodColumns := commonPodColumns
+	storeId := metav1beta1.TableColumnDefinition{
+		Name:        "StoreId",
```
Comment (weekface) [inline pkg/tkctl/readable/printers.go]:
```suggestion
		Name:        "StoreID",
```

**Comment by cofyc** [comment]
@aylei Take a look?

File: pkg/tkctl/alias/alias.go [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+// implement type Object interface
+func (t TikvList) GetObjectKind() schema.ObjectKind {
+	return t.GetObjectKind()
+}
+
+func (t TikvList) DeepCopyObject() runtime.Object {
+	return t.DeepCopyObject()
+}
+
+func (t *Tikv) FromPod(pod *v1.Pod) {
```
Comment (aylei) [inline pkg/tkctl/alias/alias.go]: consider type conversion: https://golang.org/ref/spec#Conversions

**Comment by cofyc** [comment]
https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/operator_ghpr_e2e_test_kind/detail/operator_ghpr_e2e_test_kind/289/pipeline

@xiaojingchen kind4 is not working now?

**Pull request merged**